### PR TITLE
Update testing steps.

### PR DESCRIPTION
**Pull request by wallclockbuilder**
Instructions for setting up a self-contained local Bisq network on Bitcoin regtest have been updated. 
`dev-setup.md` is deprecated. `Makefile` is new new version. 

Reference: https://github.com/bisq-network/bisq/blob/master/docs/README.md

### CODE REVIEW

**Review by ripcurlx — CHANGES_REQUESTED** [review]
NACK - path is wrong. There is not Makefile.md and I wouldn't say dev-setup.md is deprecated. There are more details in the dev-setup.md that are useful if the the default setup in the makefile is not enough for testing. I think best would be to reference both in the testing documentation.

**Comment by wallclockbuilder** [comment]
Fixed.

File: docs/testing.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
 # Bisq Testing Guide
 
+
```
Comment (ripcurlx) [inline docs/testing.md]: Could you please remove this empty line. Sometimes Travis gets stuck - in that case just leave a comment and I'll trigger a manual rebuild.

Comment (wallclockbuilder) [reply docs/testing.md]: Done. Please rebuild.

Comment (wallclockbuilder) [reply docs/testing.md]: continuous-integration/travis-ci/pr — The Travis CI build failed

Comment (ripcurlx) [reply docs/testing.md]: Just re-triggered a Travis build

File: docs/testing.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ This guide describes the testing process performed prior to each release.
 In order to take part in the testing process, you will need to do the following:
 
 - Build Bisq from source (see [build.md](build.md))
-- Setup a development/testing environment (see [dev-setup.md](dev-setup.md))
+- Setup a development/testing environment (see [Makefile](../Makefile) (new) or [dev-setup.md](dev-setup.md) (deprecated))
```
Comment (ripcurlx) [inline docs/testing.md]: As mentioned already `dev-setup.md` is not deprecated. I would rather:
```suggestion
- Setup a development/testing environment (see [Makefile](../Makefile) or [dev-setup.md](dev-setup.md))
```

**Review by ripcurlx — CHANGES_REQUESTED** [review]
NACK - Please see my comment.

**Review by ripcurlx — APPROVED** [review]
utACK

**Pull request merged**